climada/hazard/base.py

```diff
@@ -1541,6 +1543,48 @@ def local_exceedance_inten(self, return_periods=(25, 50, 100, 250)):
                    Reason: no negative intensity values were found in hazard.')
             inten_stats[inten_stats < 0] = 0
         return inten_stats
+    
+    def local_return_period(self, hazard_intensities):
```

Comment:
This is a confusing method signature to me. Why would a method of the class Hazard require the user to supply hazard intensities? These are already an attribute of the object.

Reply:
Here the variable hazard_intensities does not refer to the intensities of the Hazard object but to the threshold intensities for which the return period per grid point should be calculated. To avoid confusion we could rename this to "threshold_intensities"?

Reply:
👍 `threshold_intensities` would be much better. 😄